Description: Install documentation to /usr/share/doc
 Install docs to /usr/share/doc/luminance-hdr instead of
 /usr/share/luminance-hdr
Author: Andreas Metzler <ametzler@debian.org>
Last-Update: 2013-04-20

--- luminance-hdr-2.3.1.orig/CMakeLists.txt
+++ luminance-hdr-2.3.1/CMakeLists.txt
@@ -255,8 +255,8 @@ ELSEIF(UNIX)
     ${CMAKE_SOURCE_DIR}/README
     ${CMAKE_SOURCE_DIR}/LICENSE
     ${CMAKE_SOURCE_DIR}/Changelog
-    DESTINATION share/luminance-hdr) #info files
-    INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/help DESTINATION share/luminance-hdr) # help directory
+    DESTINATION share/doc/luminance-hdr) #info files
+    INSTALL(DIRECTORY ${CMAKE_SOURCE_DIR}/help DESTINATION share/doc/luminance-hdr) # help directory
 # Microsoft Windows
 ELSEIF(WIN32)
     # icon files to copy in the bundle
--- luminance-hdr-2.3.1.orig/cmake/Internationalization.cmake
+++ luminance-hdr-2.3.1/cmake/Internationalization.cmake
@@ -6,7 +6,7 @@ ELSEIF(UNIX)
 # ADD_DEFINITIONS(-DPREFIX=${CMAKE_INSTALL_PREFIX})
 # ADD_DEFINITIONS(-DI18NDIR="${CMAKE_INSTALL_PREFIX}/share/luminance-hdr/i18n")
 SET(I18NDIR "\"${CMAKE_INSTALL_PREFIX}/share/luminance-hdr/i18n\"")
-SET(HELPDIR "\"${CMAKE_INSTALL_PREFIX}/share/luminance-hdr/help\"")
+SET(HELPDIR "\"${CMAKE_INSTALL_PREFIX}/share/doc/luminance-hdr/help\"")
 ELSEIF(WIN32)
 # ADD_DEFINITIONS(-DI18NDIR=QCoreApplication::applicationDirPath\(\)+"/i18n")
 SET(I18NDIR QCoreApplication::applicationDirPath\(\)+"/i18n")
